### PR TITLE
change from arrays to strings

### PR DESCRIPTION
**Pull request by clayferg**

### CODE REVIEW

File: src/main/webapp/IFrame.js [hunk 1]
```diff
@@ -28,23 +28,23 @@ const vidDataRef = firestore.collection('rooms').doc(roomId)
...
+  nextThumbnail = '';
+  nextDocID = '';
+  queueDataRef.orderBy('requestTime', 'asc').limit(1).get()
       .then(function(querySnapshot) {
         querySnapshot.forEach(function(doc) {
           const queueData = doc.data();
-          docIds.push(doc.id);
-          videoIds.push(queueData.videoID);
+          nextDocID += doc.id;
+          nextVidID += queueData.videoID;
```
Comment (ciddu) [inline src/main/webapp/IFrame.js:43]: Doesn't this concatenate the video ID? Shouldn't it just be 
```

nextDocID = doc.id;
 nextVidID = queueData.videoID;
```

File: src/main/webapp/IFrame.js [hunk 2]
```diff
@@ -28,23 +28,23 @@ const vidDataRef = firestore.collection('rooms').doc(roomId)
...
           const thumbnailString = queueData.thumbnailURL;
           const thumbnailURL = thumbnailString.substring(1,
               thumbnailString.length - 1);
-          thumbnails.push(thumbnailURL);
+          nextThumbnail += thumbnailURL;
```
Comment (ciddu) [inline src/main/webapp/IFrame.js:47]: same here

File: src/main/webapp/IFrame.js (hunk 1 above)
Comment (clayferg) [reply src/main/webapp/IFrame.js:43]: Yeah, my mistake, I thought I had to concatenate to have the empty string be the default, but I now see that just setting it equal would work too the way I have it set up.

File: src/main/webapp/IFrame.js (hunk 2 above)
Comment (clayferg) [reply src/main/webapp/IFrame.js:47]: Right.

**Pull request merged**